.travis.yml

```diff
@@ -1,7 +1,23 @@
 language: r
+cache: packages
 
-r_github_packages:
-  - jimhester/covr
+matrix:
+  include:
+    - os: linux
+      r: oldrel
+    - os: linux
+      dist: trusty
```

Comment:
Does it work if we just hoist

    - dist: trusty
    - sudo: false

up to top-level? In particular, I don't think we need `sudo` for anything, meaning it's hopefully faster (or, at least, [that's what it says on the label](https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments)).